src/Clide/DevEnvInfoProvider.cs

```diff
@@ -27,7 +27,7 @@ public DevEnvInfoProvider(Lazy<IServiceLocator> serviceLocator, JoinableTaskCont
 
         DevEnvInfo GetDevEnvInfo(IServiceLocator serviceLocator)
         {
-            var vsAppId = serviceLocator.TryGetService<SVsAppId, IVsAppId>();
+            var vsAppId = serviceLocator.TryGetService<IVsAppId, IVsAppId>();
```

Comment:
And do we need both (duplicate) generic args?

Reply:
We don't. Fixed